CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+## [v1.1.0](https://github.com/bugcrowd/vulnerability-rating-taxonomy/compare/v1.0...v1.1) - 2017-04-13
+### Added
+- directory_listing_enabled
+- directory_listing_enabled.sensitive_data_exposure
+- directory_listing_enabled.non_sensitive_data_exposure
+- server_security_misconfiguration.path_traversal
+- cross_site_scripting_xss.reflected.self
+- cross_site_scripting_xss.reflected.non_self
+- cross_site_request_forgery_csrf.application_wide
+- cross_site_request_forgery_csrf.application_specific
+- cross_site_request_forgery_csrf.authenticated_action
+- cross_site_request_forgery_csrf.unauthenticated_action
+
+### Removed
+- poor_physical_security
+- social_engineering
+
+### Adjusted Priority
```

Comment:
I did not move the XSS examples that changed names to Changed as we actually added an entry with that modification. Do we draw a distinction between added/removed with consideration to a parser or expected field names? If so, the current is likely correct. 

If any changes that aren't new new entries with no corresponding removal should be listed XSS Self and Non-Self should be under Changed instead. I'm happy to abide by either convention and interested parties will have the diff to clarify the summary as well.